pkgs/by-name/bo/bounty/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  openssl,
+  libgit2,
+  pkg-config,
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "bounty";
+  version = "0.1.0-unstable-25-01-29";
+
+  src = fetchFromGitHub {
+    owner = "ghbountybot";
+    repo = "cli";
+    rev = "a1c8e74b1373ef7e0cc0cef158eeec59d56623fc";
+    hash = "sha256-sLbEGLvupDWk5FBdu4UZ4ohqMUK/HKAHhpB3oJwRA0c=";
+  };
+
+  cargoHash = "sha256-q+lnEpTsNFuQYQlQGoxZ5vnFUTqGWRMqhK+20H9Ke0I=";
+
+  doCheck = true;
```

Comment:
A `true` here will probably not work, because some of your tests use the internet and that's not allowed in the nix build sandbox.

Reply:
<img width="746" alt="image" src="https://github.com/user-attachments/assets/74fd7e92-602d-48af-a3c8-70ea975b1576" />

none of my tests use the internet. This is me running them with WiFi disabled.

Reply:
That is good. I had tried an earlier version where some of the tests failed, so I jumped to conclusions.